api/s2n.h

```diff
@@ -596,6 +596,269 @@ extern int s2n_cert_get_utf8_string_from_extension_data_length(const uint8_t *ex
 S2N_API 
 extern int s2n_cert_get_utf8_string_from_extension_data(const uint8_t *extension_data, uint32_t extension_len, uint8_t *out_data, uint32_t *out_len);
 
+/* Pre-shared key (PSK) Hash Algorithm - RFC 8446 Section-2.2 */
+/* https://tools.ietf.org/html/rfc8446#section-2.2 */
```

Comment:
Nit: Why is this two block comments instead of one? I also wouldn't bother citing the RFC section when you're already linking to it.

Reply:
I am following the current format used for enum description - I can get rid of it thought.